platform/features/fixed/templates/fixed.html

```diff
@@ -44,15 +44,15 @@
         </mct-include>
     </div>
     <!-- Selection highlight, handles -->
-    <span class="s-selected s-moveable" ng-if="controller.isElementSelected()">
-        <div class="l-fixed-position-item t-edit-handle-holder"
+    <span class="c-frame-edit" ng-if="controller.isElementSelected()">
+        <div class="c-frame-edit__move"
              mct-drag-down="controller.moveHandle().startDrag()"
              mct-drag="controller.moveHandle().continueDrag(delta)"
              mct-drag-up="controller.endDrag()"
              ng-style="controller.getSelectedElementStyle()">
         </div>
         <div ng-repeat="handle in controller.handles()"
-             class="l-fixed-position-item-handle edit-corner"
+             class="c-frame-edit__handle --nwse"
```

Comment:
what is --nwse?  @charlesh88 @psarram

Reply:
@larkin It's defined in _global.scss and sets the resize cursor.